core/src/main/java/org/pentaho/di/connections/vfs/provider/ConnectionFileNameUtils.java

```diff
@@ -0,0 +1,60 @@
+package org.pentaho.di.connections.vfs.provider;
+
+import edu.umd.cs.findbugs.annotations.NonNull;
+import edu.umd.cs.findbugs.annotations.Nullable;
+import org.apache.commons.lang.StringUtils;
+
+import static org.pentaho.di.connections.vfs.provider.ConnectionFileObject.DELIMITER;
+import static org.pentaho.di.connections.vfs.provider.ConnectionFileObject.DELIMITER_CHAR;
+
+public class ConnectionFileNameUtils {
```

Comment:
I was writing these connection file name related utilities in more than one class, so just created this helper class to centralize the code.